src/elf.rs

```diff
@@ -379,16 +379,18 @@ impl<C: ContextObject> Executable<C> {
                 .ok_or(ElfParserError::OutOfBounds)?,
         );
         let config = loader.get_config();
-        let sbpf_version = if config.enabled_sbpf_versions.end() == &SBPFVersion::V1 {
+        let sbpf_version = if config.enabled_sbpf_versions.end() == &SBPFVersion::V0 {
             if e_flags == EF_SBPF_V2 {
-                SBPFVersion::V2
+                SBPFVersion::Reserved
             } else {
-                SBPFVersion::V1
+                SBPFVersion::V0
```

Comment:
If we get a 3 in the `e_flags`, the version will be `V0` here.

Reply:
Yep exactly, that is the bug we are feature gating out here.